binder/src/main/java/io/grpc/binder/internal/BinderTransportSecurity.java

```diff
@@ -60,15 +63,15 @@ public static void installAuthInterceptor(ServerBuilder<?> serverBuilder) {
    *
    * @param builder The {@link Attributes.Builder} for the transport being created.
    * @param remoteUid The remote UID of the transport.
-   * @param securityPolicy The policy to enforce on this transport.
+   * @param serverPolicyChecker The policy enforcer for this transport.
```

Comment:
maybe "enforcer" -> "checker" since it's actually this class which enforces the policy.